src/main/java/robot/MainTest.java

```diff
@@ -0,0 +1,13 @@
+package robot;
```

Comment:
Why isn't this file inside the `pure_pursuit` package?

Reply:
Will cause conflicts with calculate branch, MainTest was only used there so i think its better to make this change there.